### DISCUSSION TITLE
Unable to see  zkSync Sepolia on metamask

### DISCUSSION THREAD

**Discussion by nwoyiejames** [Cyfrin/foundry-full-course-cu#2323] [General]
please, i was unable to add zkSync Sepolia to my metamask ,because it was listed in the add network .
How can I go about  to  the zkSync Sepolia testnet

**Comment by EngrPips** [comment]
Hello @nwoyiejame, are you trying to add `zksync sepolia` testnet to your metamask through `chainlist`?

**Comment by Haardtripathi** [comment]
Did you bridge funds from your Ethereum Sepolia testent to the Zksync sepolia testnet using https://portal.zksync.io/bridge?network=sepolia?

**Comment by ArnaudBand** [comment]
Hi @nwoyiejames, if you want to add zkSync visit this link. You active the testnet search zksync [chainlist](https://chainlist.org/). You will connect your wallet and add it.

**Reply by EngrPips** [reply]
Awesome. Thanks for the heads-up.
